abelfunctions/differentials.py

```diff
@@ -112,7 +112,16 @@ def mnuk_conditions(g, b, generic_adjoint):
     r = r_reduced_mod_denom(u)  # first need to coerce to "largest" ring, T
```

Comment:
I was just looking issue at this the other day, and I think this could be a simpler, though not necessarily better, solution:
`r = T(r_reduced_mod_denom(u))`

Reply:
Thanks!  That does seem to be a more elegant solution so I've made that change instead.